__tests__/materials/material.mutation.test.js

```diff
@@ -1,427 +1,415 @@
+                }
+              }
+              additionalPrice {
+                currency
+                value
               }
+              available
             }
-          `,
-        })
-        .catch(e => e);
-      const newMaterial = res.data.addMaterial;
-      materialId = newMaterial._id;
+            ... on Error {
+              message
+              statusCode
+            }
+          }
+        }
+      `,
+      variables: { material },
+    });
 
-      expect(newMaterial).toBeDefined();
-      expect(newMaterial).toHaveProperty(
-        'name',
-        material.name.map(item => ({ ...languageTypeName, ...item }))
-      );
-      expect(newMaterial.name).toBeInstanceOf(Array);
+    const addedMaterial = res.data.addMaterial;
+    materialId = res.data.addMaterial._id;
+    expect(addedMaterial).toBeDefined();
 
-      expect(newMaterial).toHaveProperty(
-        'description',
-        material.description.map(item => ({ ...languageTypeName, ...item }))
-      );
-      expect(newMaterial.description).toBeInstanceOf(Array);
+    expect(addedMaterial).toHaveProperty(
+      'name',
+      material.name.map(item => ({
+        ...item,
+      }))
+    );
+    expect(addedMaterial.name).toBeInstanceOf(Array);
 
-      expect(newMaterial).toHaveProperty('purpose', material.purpose);
+    expect(addedMaterial).toHaveProperty(
+      'description',
+      material.description.map(item => ({
+        ...item,
+      }))
+    );
+    expect(addedMaterial.description).toBeInstanceOf(Array);
 
-      expect(newMaterial).toHaveProperty('available', material.available);
+    expect(addedMaterial).toHaveProperty('purpose', material.purpose);
+    expect(addedMaterial).toHaveProperty('available', material.available);
 
-      expect(newMaterial).toHaveProperty(
-        'additionalPrice',
-        material.additionalPrice.map(item => ({
-          ...currencyTypeName,
-          ...item,
-        }))
-      );
-      expect(newMaterial.additionalPrice).toBeInstanceOf(Array);
+    expect(addedMaterial).toHaveProperty(
+      'additionalPrice',
+      material.additionalPrice.map(item => ({
+        ...item,
+      }))
+    );
+    expect(addedMaterial.additionalPrice).toBeInstanceOf(Array);
 
-      expect(newMaterial).toHaveProperty('colors', [
-        {
-          __typename: 'Color',
-          code: 777,
-          name: material.colors[0].name.map(item => ({
-            ...languageTypeName,
-            ...item,
-          })),
-          images: { ...imageTypeName, ...material.colors[0].images },
-          available: true,
-          simpleName: material.colors[0].simpleName.map(item => ({
-            ...languageTypeName,
-            ...item,
-          })),
+    expect(addedMaterial).toHaveProperty('colors', [
+      {
+        code: 777,
+        name: material.colors[0].name.map(item => ({
+          ...item,
+        })),
+        simpleName: material.colors[0].simpleName.map(item => ({
```

Comment:
Please, simplify it to the `material.colors[0].simpleName` because mapping is useless now due to the returning the same object